src/core_atmosphere/physics/physics_wrf/module_mynnsfclay.F90

```diff
@@ -600,7 +600,7 @@ subroutine mynnsfclay(                                 &
           regime(i)=2.
        endif
 
-       if (.not. restart_or_cycle .or. (restart_or_cycle .and. itimestep > 1) ) then
+       if (.not. restart_or_cycle .and. itimestep .le. 1) then
```

Comment:
The change looks good with me.
Just curious, do we have situations when `itmestep` < 1? 
I saw `if (itimestep == 1) then` was alos used before PR #136 

<img width="910" height="377" alt="image" src="https://github.com/user-attachments/assets/96526b4d-1fc3-4b83-be80-afd7ce7fd57e" />

Reply:
I don't think so. I think itimestep starts at 1 (on the first time step) in mpas_atm_core.F and is incremented there too.

I do see some places in the code where "if(itimestep > 0) then", which implies otherwise, but I don't see where it can get set to 0 (or negative).